packages/addons/addon-depends/inadyn/libconfuse/patches/gettext-0.20-libconfuse.patch

```diff
@@ -0,0 +1,11 @@
+--- libconfuse-3.3/configure.ac	2020-06-25 05:21:59.000000000 +0000
++++ libconfuse-3.3/configure.ac	2020-12-29 02:04:36.074750823 +0000
+@@ -29,7 +29,7 @@
+ # Unfortunately CentOS7, RHEL7 ships 0.18.2.1, so for best compat.
+ # level at this point in time we set 0.18.2.
+ AM_GNU_GETTEXT([external])
+-AM_GNU_GETTEXT_VERSION([0.18.2])
++AM_GNU_GETTEXT_VERSION([0.20])
```

Comment:
can we do it that way
https://github.com/LibreELEC/LibreELEC.tv/pull/4833/files
that would be upstream able too

Reply:
Patch updated.